Makefile

```diff
@@ -49,6 +49,66 @@ endif
 receptor: $(shell find pkg -type f -name '*.go') ./cmd/receptor-cl/receptor.go
 	CGO_ENABLED=0 GOFLAGS="-buildvcs=false" go build -o receptor $(DEBUGFLAGS) -ldflags "-X 'github.com/ansible/receptor/internal/version.Version=$(VERSION)'" $(TAGPARAM) ./cmd/receptor-cl
 
+clean:
```

Comment:
why move this?

Reply:
alphabetizing so we can find targets faster.  We need to reorder all of them in a future PR